### PR TITLE
Model ensembles

### PR DESCRIPTION
**Pull request by JamiesonWarner**

### CODE REVIEW

File: covid_xprize/examples/predictors/conditional_lstm/train_predictor.py [hunk 1]
```diff
@@ -65,6 +66,7 @@ def train_predictor(training_data: pd.DataFrame,
     :param context_column: Which column in the data df to use as context and outcome.
     :param arch: Which predictor architecture to use.
         Current options are 'conditional' and 'independent'.
+    :param return_all_trials: If set to True, then this function returns all trials as a list.
```
Comment (ofrancon) [inline covid_xprize/examples/predictors/conditional_lstm/train_predictor.py:69]: A "trial" is a trained model in that case? Maybe worth mentioning it's going to return "all trials as a list of trained models"

File: requirements.txt [hunk 2]
```diff
@@ -5,8 +5,8 @@ numpy==1.24.2
 notebook==6.5.3
 scikit-learn==1.2.2
 scipy==1.10.1
-tensorflow==2.11.1
-keras==2.11.0
+tensorflow==2.13.0
+keras==2.13.1
```
Comment (ofrancon) [inline requirements.txt:9]: Thanks for updating these deps. One reason it's needed is for people using Python 3.11: tensorflow 2.11.1 has not been released for it. Only tensorflow 2.12 and onward have been release for python 3.11.
`setup.py` has to be updated too, though. As well as LIBRARY_VERSION = '2.0.2' there, to 2.0.3

**Review by ofrancon — CHANGES_REQUESTED** [review]
Please update `setup.py`too:
- keras and tensorflow versions: they should match requirements.txt
- LIBRARY_VERSION = '2.0.3'
And then feel free to release version 2.0.3

File: covid_xprize/examples/predictors/conditional_lstm/train_predictor.py (hunk 1 above)
Comment (JamiesonWarner) [reply covid_xprize/examples/predictors/conditional_lstm/train_predictor.py:69]: 👍

File: requirements.txt (hunk 2 above)
Comment (JamiesonWarner) [reply requirements.txt:9]: 👍

**Review by ofrancon — APPROVED** [review]
lgtm, thanks!

**Pull request merged**